user/deployment/releases.md

```diff
@@ -39,6 +39,15 @@ Or, if you're using a private repository:
 travis setup releases --pro
 ```
 
+If you are using the [`branches.only` property](/user/customizing-the-build#Building-Specific-Branches), remember that when you push a tag, the [`$TRAVIS_BRANCH` property](/user/environment-variables/#Default-Environment-Variables) contains the name of the tag. As a result, edit the `branches.only` property to add the names of the tags you might push in the future. You can use a regular expression if you have formalized names. For example: 
+
+```yaml
+   branches:
+    only:
+    - master
+    - /^v.*$/
+```
+
```

Comment:
Thanks for this. It is probably worth explaining the intent of the regular expression for people who aren't familiar with them?

Reply:
Sure. I added a quick explanation. I imagine that people tinkering with Travis already know about regexp :)

Reply:
You might be surprised. I think the new explanation suffices though - thank you!